java/src/main/resources/template.properties

```diff
@@ -242,6 +242,20 @@ kafka.bq.fail.on.dataloss=false
 ## Ouptut mode for writing data. Accepted values: 'append', 'complete', 'update'
 kafka.bq.stream.output.mode=append
 
+
+#Kafka Common
+kafka.message.format=
+kafka.bootstrap.servers=
+kafka.topic=
+kafka.starting.offset=
+kafka.schema.url=
+
+#Kafka To GCS
+kafka.gcs.output.location=;
```

Comment:
why location has `;` ?

Reply:
Typo fixed. Will push the changes and resolve the conversation